### DISCUSSION TITLE
Request for Next.js.13 (App) router + Prisma documentation

### DISCUSSION THREAD

**Discussion by GavinSalcedo** [prisma/prisma#20762] [Q&A] [answered]
### Question

Is there an existing documentation for Next.js using the app directory integrating Prisma?

I understand this should be straightforward for Next.js developers to understand and migrate from pages to app, however, I think it would also benefit other developers if given more reference.

App router is becoming more stable and evidently effective than pages, so I would like to request to update the Prisma website documentation.

Thanks for the considering this post.

reference: https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/fullstack

### How to reproduce (optional)
Click on this link to access the existing documentation
- https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/fullstack

_No response_

**Comment by ludralph** [comment]
Hi @GavinSalcedo 👋 

Thank you for raising this request.  I have shared it internally with the team.

**Reply by GavinSalcedo** [reply]
Thanks @ludralph looking forward to it!

**Reply by ludralph** [reply, marked as answer]
Hi @GavinSalcedo, Did you get a chance to see this [example](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) that uses the `App` directory and does this solve your issue?

**Reply by GavinSalcedo** [reply]
@ludralph yes it does! Would strongly suggest you put this reference on your web documentation also.

**Reply by iqbal125** [reply]
This seems like a incorrect implementation. Its using both the app and pages directory at the same time, which is wrong. 

Im going to create an example project of this myself  and post it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
